### PR TITLE
PokeAPI: Unpin connector

### PR DESCRIPTION
**Pull request by flash1293**
Unpin the PokeAPI connector

### CODE REVIEW

**Comment by airbyte-oss-build-runner** [comment]
##  source-pokeapi test report (commit [`65c77d49ec`](https://github.com/airbytehq/airbyte/pull/31378/commits/65c77d49eceeb696dda510d43b55ec323065a677)) - ✅

⏲️  Total pipeline duration: 01mn28s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-pokeapi docker image for platform(s) linux/x86_64 | ✅        |
| Acceptance tests                                               | ✅        |
| Code format checks                                             | ✅        |
| Validate metadata for source-pokeapi                           | ✅        |
| Connector version semver check                                 | ✅        |
| Connector version increment check                              | ✅        |
| QA checks                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_unpine-pokeapi/1697193784/65c77d49eceeb696dda510d43b55ec323065a677/source-pokeapi/0.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:65c77d49eceeb696dda510d43b55ec323065a677)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pokeapi test
```

File: docs/integrations/sources/pokeapi.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ The PokéAPI uses the same [JSONSchema](https://json-schema.org/understanding-js
 
 | Version | Date       | Pull Request                                             | Subject                                         |
 | :------ | :--------- | :------------------------------------------------------- | :---------------------------------------------- |
-| 0.2.0   | 2023-10-02 | [30969](https://github.com/airbytehq/airbyte/pull/30969)   | Migrated to Low code
-                                   |
+| 0.2.1   | 2023-10-13 | [31378](https://github.com/airbytehq/airbyte/pull/31378)   | Unpin version                                 |
```
Comment (girarda) [inline docs/integrations/sources/pokeapi.md:39]: why do we need to publish a new version? can't we just unpin so 0.2.0 is used?

**Comment by flash1293** [comment]
@girarda I thought that was necessary to make changes to the metadata yaml, sorry. will adjust

**Comment by airbyte-oss-build-runner** [comment]
##  source-pokeapi test report (commit [`d7cef1318b`](https://github.com/airbytehq/airbyte/pull/31378/commits/d7cef1318b0d50be7cdfdd05984751fd6c68aeb5)) - ✅

⏲️  Total pipeline duration: 49.61s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-pokeapi docker image for platform(s) linux/x86_64 | ✅       |
| Acceptance tests                                               | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-pokeapi                           | ✅       |
| Connector version semver check                                 | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_unpine-pokeapi/1697447933/d7cef1318b0d50be7cdfdd05984751fd6c68aeb5/source-pokeapi/0.2.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:d7cef1318b0d50be7cdfdd05984751fd6c68aeb5)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pokeapi test
```

**Review by girarda — APPROVED** [review]

**Pull request merged**